### PR TITLE
Clean up Fetch Connector

### PR DESCRIPTION
**Pull request by elhansen748**
Removing references to non-existent sample connector and cleaning up links

## Pull Request Checklist

**Complete and check all items that are applicable.**

- [ ] Proofread documentation changes for spelling and grammatical errors
- [ ] Used Markdown code blocks for all applicable examples using code
- [ ] Used relative links when linking to other documentation on Developer Center
- [ ] If creating new API reference pages, followed the guidelines stated [in the wiki](https://github.com/concur/developer.concur.com/wiki/Creating-API-Reference-pages)
- [ ] If moving pages, added [page redirects](https://github.com/concur/developer.concur.com/wiki/Adding-Redirects) so users are redirected to the new page

### CODE REVIEW

**Comment by elhansen748** [comment]
Hey Brian, made the change

File: src/api-reference/callouts/fetch-attendee.markdown [hunk 1]
```diff
@@ -106,10 +104,7 @@ The Concur administrator can select which attendee types use the connector when
 
 **Professional/Premium only:** If desired, the administrator can create a new attendee type specifically for use with the connector.
 
-### Responses and Errors
-Refer to the HTTP Codes page for details of the common responses and errors.
-
 
-[1]: /docs/guides/REST-API-Basics.html
+[1]: /api-reference/callouts/callouts-application-connectors.html
```
Comment (bhague1281) [inline src/api-reference/callouts/fetch-attendee.markdown]: Can you also remove the trailing spaces at the end of this line?

File: src/api-reference/callouts/fetch-attendee.markdown [hunk 2]
```diff
@@ -106,10 +104,7 @@ The Concur administrator can select which attendee types use the connector when
...
 
-### Responses and Errors
-Refer to the HTTP Codes page for details of the common responses and errors.
-
 
-[1]: /docs/guides/REST-API-Basics.html
+[1]: /api-reference/callouts/callouts-application-connectors.html  
 [2]: http://forum.developer.concur.com/
-[5]: /api-reference/expense/attendees/index.html
+[5]: /api-reference/callouts/post-fetch-attendee.html
```
Comment (bhague1281) [inline src/api-reference/callouts/fetch-attendee.markdown]: Can you also remove the trailing spaces at the end of this line?

**Comment by elhansen748** [comment]
Ok, trailing spaces should be removed now

**Pull request merged**